### PR TITLE
Move pattern routes to sponged

### PR DESCRIPTION
**Pull request by dwhitena**
- move `pattern` CRUD routes to `sponged` as they only relate to import.
- update `xenia` and `sponge` CLI tools.

### CODE REVIEW

**Comment by jde** [comment]
Agree that these belong here. As we're making this shift, I'd like to raise a terminology question.

Currently, the term patterns makes sense because the only content of these pieces of config are a type's relationships with other types.

We will soon need, however, to add more options to the config that describes items, including validation rules in the short term and potentially version migration strategies in the long run.

Long story short, I propose renaming "pattern" to "schema" as it is a higher scoped concept that will include all the things we need to set about an item by type.

**Comment by dwhitena** [comment]
@jde Yes, I'm ok with changing the name if you think it makes sense.  I would suggest avoiding the term `schema` just because of the baggage associated with it.  Would you like me to make this change for this PR or just open some discussion around the name?

**Comment by jde** [comment]
Good thought on not using terms with baggage. Let's merge this and have a larger discussion about configuration and schema in the coming weeks. We may want to make other config based adjustments once we get the feel for this system. Let's do them all at once.
